request.go

```diff
@@ -321,7 +321,23 @@ func unpackRequestBody(buf []byte, reqObj interface{}) error {
 	if err != nil {
 		return perrors.WithStack(err)
 	}
-	req[6] = attachments
+	if v, ok := attachments.(map[interface{}]interface{}); ok {
+		req[6] = ToMapStringString(v)
+	} else {
+		return perrors.Errorf("get wrong attachments: %+v", attachments)
+	}
 
 	return nil
 }
+
+func ToMapStringString(origin map[interface{}]interface{}) map[string]string {
+	dest := make(map[string]string)
+	for k, v := range origin {
+		if kv, ok := k.(string); ok {
+			if vv, ok := v.(string); ok {
+				dest[kv] = vv
```

Comment:
none string key/value will be ignored, is that expected?

Reply:
This is to prevent `panic `from happening.  and It's impossible to k/v that is `string` theoretically.
Or, you think It's better how to deal with it.

Reply:
skip the review if that is what u want